validator/resources/healthri/config.properties

```diff
@@ -1,11 +1,37 @@
-validator.type = review
+validator.type = v1.0.0, development
```

Comment:
Do we dare to add a `latest` tag here, linking to https://github.com/Health-RI/health-ri-metadata/releases/latest? That would assume that the shapes are attached as artifacts to the release.

Reply:
I tried that (and would be the best solution) but 'latest; only points to latest release zip file.